api/src/utils/csv-utils/csv-config-utils.ts

```diff
@@ -204,26 +204,79 @@ export class CSVConfigUtils<StaticHeaderType extends Uppercase<string> = Upperca
     }
   }
 
+  /**
+   * Get the array-cell values from a CSV row.
+   *
+   * @param {StaticHeaderType} header - The header name
+   * @param {CSVRow} row - The CSV row
+   * @param {{ delimiter: string }} options - The options
+   * @return {(string[] | undefined)} - The array-cell values or undefined if none found
+   */
+  getArrayCellValue(header: StaticHeaderType, row: CSVRow, options: { delimiter: string }): string[] | undefined {
+    let cellValue: CSVCell;
+
+    // Static header or dynamic header exact match
+    if ((header as Uppercase<string>) in row) {
+      cellValue = row[header];
+    }
+
+    // Attempt to find the array-cell values from the header aliases
+    for (const alias of this.config.staticHeadersConfig[header]?.aliases ?? []) {
+      cellValue = row[alias];
+    }
+
+    if (!cellValue) {
+      // No cell value found
+      return undefined;
+    }
+
+    // Split the original cell value by the delimiter, trimming whitespace and removing empty values
+    return String(cellValue)
+      .split(options.delimiter)
+      .map((value) => value.trim())
+      .filter((value) => value.length > 0);
+  }
+
   /**
    * Get all the cell values from a static header.
    *
    * @param {StaticHeaderType} header - The header name
-   * @returns {any[]} - The cell values
+   * @returns {CSVCell[]} - The cell values
    */
   getCellValues(header: StaticHeaderType): CSVCell[] {
     return this.worksheetRows.map((row) => this.getCellValue(header, row));
   }
 
+  /**
+   * Get all the array-cell values from a static header.
+   *
+   * @param {StaticHeaderType} header - The header name
+   * @return {((string | undefined)[])} - The array-cell values
+   */
+  getArrayCellValues(header: StaticHeaderType): (string | undefined)[] {
+    return this.worksheetRows.flatMap((row) => this.getArrayCellValue(header, row, { delimiter: ';' }));
```

Comment:
question: Should the delimiter be hardcoded?

Reply:
fixed